examples/elasticsearch/es-rc.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: es
+  namespace: default
+  labels:
+    component: elasticsearch
+spec:
+  replicas: 1
+  selector:
+    component: elasticsearch
+  template:
+    metadata:
+      labels:
+        component: elasticsearch
+    spec:
+      serviceAccount: elasticsearch
+      containers:
+      - name: es
+        image: quay.io/pires/docker-elasticsearch-kubernetes:1.7.1-4
+        env:
+        - name: KUBERNETES_TRUST_CERT
+          value: "true"
```

Comment:
So tried removing this & adding the following 2 env vars:

```
- name: KUBERNETES_CA_CERTIFICATE_FILE                                     
  value: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt              
- name: KUBERNETES_SERVICE_HOST                                            
  value: kubernetes.default.svc
```

means it can talk securely to the master for discovery. This is fixed in later versions of the client, but no ES discovery plugin released with that version yet.